util/dvsim/utils.py

```diff
@@ -180,6 +180,13 @@ def find_and_substitute_wildcards(sub_dict, full_dict, ignored_wildcards=[]):
 def md_results_to_html(title, css_path, md_text):
     '''Convert results in md format to html. Add a little bit of styling.
     '''
+    # TODO: make sure that browsers do not cache this site.
+    # need to check what method works for our server setup.
+    # html_text  = '<?php\n'
+    # html_text += 'header("Cache-Control: no-cache, no-store, must-revalidate");\n'
+    # html_text += 'header("Pragma: no-cache");\n'
+    # html_text += 'header("Expires: 0");\n'
+    # html_text += '?>\n'
```

Comment:
I added this comment since I experienced cases where files did not update upon reloading until I deleted my browser's cache... but maybe the effect of what you describe is not instantaneous (or it's getting cached somewhere along the way).